web/packages/core/src/ruffle-player.ts

```diff
@@ -230,6 +230,29 @@ export class RufflePlayer extends HTMLElement {
     constructor() {
         super();
 
+        const resizeObserver = new ResizeObserver((entries) => {
+            const heightAttr = this.attributes.getNamedItem("height");
+            const parentHeight = this.parentElement?.clientHeight;
+            if (heightAttr !== undefined && heightAttr !== null) {
+                const height = RufflePlayer.htmlDimensionToCssDimension(
+                    heightAttr.value,
+                );
+
+                if (
+                    parentHeight &&
+                    parentHeight > 0 &&
+                    height?.endsWith("%") &&
+                    entries.length > 0 &&
+                    entries[0] &&
+                    entries[0].target?.clientHeight === 0
+                ) {
+                    this.dynamicStyles.sheet?.insertRule(
+                        `:host { height: 200px !important; }`,
```

Comment:
Should this ever remove the rule?

Reply:
The rule is removed in `updateStyles` along with all other dynamic styles.

Reply:
That said looking at it I don't think setting CSS width and height styles with javascript currently works at all even without this PR.